[bitnami/redis] Redis chart has invalid priorityClassName

<!--
 Before you open the bug report please review the following troubleshooting guide:

 - [Troubleshoot Bitnami Helm Chart Issues](https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues)
 -->

**Which chart**: bitnami/redis

**Describe the bug**

The helm chart has a default for the priorityClassName as `{}` where it should be `nil` instead.

```bash
helm upgrade redis bitnami/redis
# coalesce.go:196: warning: cannot overwrite table with non table for priorityClassName (map[])
```

My chart values use:

```yaml
master:
  priorityClassName: high-priority
slave:
  priorityClassName: high-priority
```

**Additional context**

This was fixed at some point (see issue #3997) that was then re-introduced https://github.com/bitnami/charts/commit/3a232d78e33b4e5a4a332114534e38e502eab766

--- a/bitnami/redis/Chart.yaml
+++ b/bitnami/redis/Chart.yaml
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 12.9.1
+version: 12.9.2

--- a/bitnami/redis/values.yaml
+++ b/bitnami/redis/values.yaml
@@ -490,7 +490,7 @@ master:
 
   ## Redis(TM) Master pod priorityClassName
   ##
-  priorityClassName: {}
+  priorityClassName: null
 
   ## An array to add extra env vars
   ## For example:
@@ -654,7 +654,7 @@ slave:
   podAnnotations: {}
 
   ## Redis(TM) slave pod priorityClassName
-  priorityClassName: {}
+  priorityClassName: null
 
   ## Enable persistence using Persistent Volume Claims
   ## ref: http://kubernetes.io/docs/user-guide/persistent-volumes/
@@ -832,7 +832,7 @@ metrics:
     rules: []
 
   ## Metrics exporter pod priorityClassName
-  priorityClassName: {}
+  priorityClassName: null
   service:
     type: ClusterIP
 

